OWNERS.md

```diff
@@ -122,3 +124,7 @@ matter expert reviews. Feel free to loop them in as needed.
   * v2 xDS, listeners, filter chain discovery service.
 * Michael Payne ([moderation](https://github.com/moderation)) (m@m17e.org)
   * External dependencies, Envoy's supply chain and documentation.
+* Cerek Hillen ([crockeo](https://github.com/crockeo)) (chillen@lyft.com)
+  * Python and C++ platform bindings.
+* Charles Le Borgne ([carloseltuerto](https://github.com/carloseltuerto)) (cleborgne@google.com)
```

Comment:
I think sadly Charles counts as Emeritus

Reply:
Cerek too probably then.

Reply:
Kept Cerek in friends 😄